### PR TITLE
docs: button docs improvements

### PR DESCRIPTION
**Pull request by mikerodonnell89**
part of #2045 

Improves the look of button docs

### CODE REVIEW

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
It's not applied on all of the buttons:
Screen Shot 2021-01-13 at 4 33 40 PM
Screen Shot 2021-01-13 at 4 33 25 PM
Screen Shot 2021-01-13 at 4 33 18 PM
Screen Shot 2021-01-13 at 4 33 06 PM

File: .storybook/custom/custom.scss [hunk 1]
```diff
@@ -286,6 +286,15 @@ code.docs-code-grey {
   background-color: #f8f8f8;
 }
 
+.fddocs-button-container {
```
Comment (InnaAtanasova) [inline .storybook/custom/custom.scss:289]: Won't this work if you apply it on the `fddocs-container` without adding additional class? And prob should be `fd-docs-container`?

Comment (mikerodonnell89) [reply .storybook/custom/custom.scss:289]: `fddocs-container` was already in place so I just kept it

**Comment by JKMarkowski** [comment]
For me it works fine, but not for split button, which I guess has stronger selector:
image

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**